packages/axios-extension/src/abap/adt-catalog/generators/ui-service-generator.ts

```diff
@@ -124,6 +124,6 @@ export class UiServiceGenerator extends AdtService {
                 corrNr: transport
             }
         });
-        return JSON.parse(response.data);
+        return this.parseResponse(response.data);
```

Comment:
@docirl, in line 120 above we set `Accept` to `+json` and `+xml`. With this change we handle only xml, no json anymore. Are we sure the response is always xml?

Reply:
Yes the response is json, and response.data is an xml string.